map/mwm_url.cpp

```diff
@@ -126,7 +126,7 @@ void ParsedMapApi::AddKeyValue(string key, string const & value, vector<ApiPoint
     if (!strings::to_double(value, m_zoomLevel))
       m_zoomLevel = 0.0;
   }
-  else if (key == "n")
+  else if (key == "n" || key == "saddr" || key == "daddr")
```

Comment:
What these magic string literals mean? How about to create documented named constants for them?